base/uk.ac.stfc.isis.ibex.ui.motor/src/uk/ac/stfc/isis/ibex/ui/motor/views/MinimalMotorViewModel.java

```diff
@@ -255,6 +293,17 @@ public void setEnabled(MotorEnable enabled) {
             this.enabled = false;
         }
     }
+    
+    private void refreshLabels() {
+    	setSetpoint(formatForMotorDisplay("SP", motor.getSetpoint().getSetpoint()));
```

Comment:
I think you want `getX` and `setX` to be of the same type otherwise it's a bit confusing (especially for databinding/property changes). At the moment both `get` and `set` are `String`, if we do it as you suggest I think we would have `set` as `Double` and `get` still as `String`?

If I have misunderstood what you meant here please shout